src/sailfish-iptables-dbus.c

```diff
@@ -1203,32 +1227,31 @@ gint sailfish_iptables_dbus_unregister()
 	gint rval = 0;
 
 	DBusConnection* conn = connman_dbus_get_connection();
-	if(conn)
-	{
+	if(conn) {
 		// First send the signal to all
 		DBusMessage *signal = sailfish_iptables_dbus_signal(
 					SAILFISH_IPTABLES_SIGNAL_STOP,
 					DBUS_TYPE_INVALID, NULL);
 		if(signal)
 			sailfish_iptables_dbus_send_signal(signal, NULL);
-			
+
 		if(!g_dbus_unregister_interface(conn,
 			SAILFISH_IPTABLES_DBUS_PATH,
-			SAILFISH_IPTABLES_DBUS_INTERFACE))
-		{
-			DBG("%s %s %s", PLUGIN_NAME, "sailfish_iptables_dbus_unregister():",
-				"unregsiter failed");
+			SAILFISH_IPTABLES_DBUS_INTERFACE)) {
+			DBG("%s %s", PLUGIN_NAME,
+				"sailfish_iptables_dbus_unregister(): "
+				"unregister failed");
 			rval = 1;
 		}
 		dbus_connection_unref(conn);
 	}
-	else 
+	else
 	{
-		DBG("%s %s","sailfish_iptables_dbus_unregister():",
-			"no dbus connection");
+		DBG("%s","sailfish_iptables_dbus_unregister(): no dbus "
+			"connection");
```

Comment:
I would keep the "no dbus " in the next last line for readability.

Reply:
Well, yeah, might be better like that. Changed back in fa991f8b2b35dab6f4dae2b1624c088f2b99d37d